play/src/pusher/enums/EnvironmentVariableValidator.ts

```diff
@@ -150,6 +150,7 @@ export const EnvironmentVariables = z.object({
     MATRIX_PUBLIC_URI: z.string().optional(),
 
     EMBEDLY_KEY: z.string().optional(),
+    CHAT_TYPE: z.union([z.literal("PROXIMITY"), z.literal("MATRIX")]).default("PROXIMITY"),
```

Comment:
If we use an env var to switch between one chat type and the other, we could use a discriminating union.

This way, if `CHAT_TYPE = "MATRIX"`, then all the matrix parameters would be compulsory and not optional?